### PR TITLE
fix benchmarks

### PR DESCRIPTION
**Pull request by klahnakoski**
This is to fix the lack of benchmark data on quantum pages

### CODE REVIEW

**Comment by klahnakoski** [comment]
Changing the signatures was not enough to get the charts to work.  The code assumed `framework==1`, so that parameter had to be weaved throughout.  Some simplification of the subbenchmark code was also done.

File: README.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ Use [Perfherder](https://treeherder.mozilla.org/perf.html#/graphs) directly to d
 Pass a `series` array with each element describing one of your data series. To define a series you need these parameters:
 
 * label - It will be the legend value
-* frameworkId - 1 for Talos, 10 for Raptor and 11 for js-bench.
+* framework - 1 for Talos, 10 for Raptor and 11 for js-bench.
```
Comment (armenzg) [inline README.md]: I've been trying to keep the same nomenclature, however, I might have chosen the internal variable name.
Treeherder's usage of [frameworkId](https://github.com/mozilla/treeherder/search?q=frameworkId&unscoped_q=frameworkId).

File: src/quantum/config.js [hunk 2]
```diff
@@ -1,66 +1,76 @@
...
+    //   },
+    // },
+    // {
+    //   title: 'JetStream',  //RUN ON JS BENCH
+    //   secondLink: '/quantum/windows10-64/JetStream',
+    //   secondTitle: 'Breakdown',
+    //   signatures: {
+    //     'windows10-64': '799db0d801c7248924e93ca5a0b770020498da69',
+    //   },
+    // },
```
Comment (armenzg) [inline src/quantum/config.js]: Please remove commented out code.

File: src/quantum/config.js [hunk 3]
```diff
@@ -1,66 +1,76 @@
...
+    //   secondLink: '/quantum/windows7-32/ARES6',
+    //   secondTitle: 'Breakdown',
+    //   signatures: { 'windows7-32': '9585321f110f16414e272956c8892a38d03e7797' },
+    // },
+    // {
+    //   title: 'JetStream',  //RUN ON JS BENCH
+    //   secondLink: '/quantum/windows7-32/JetStream',
+    //   secondTitle: 'Breakdown',
+    //   signatures: { 'windows7-32': '95535e0278558bd59ae324920ca0469278faadb4' },
+    // },
```
Comment (armenzg) [inline src/quantum/config.js]: Same here.

File: src/utils/perfherder/subbenchmarks.jsx [hunk 4]
```diff
@@ -12,70 +14,65 @@ const subtests = async signatureHash => {
...
-      } else {
-        delete suites[elem.option_collection_hash];
-      }
-    }
-  });
+  const optionHashes = frum(options)
+    .where({ 'options.0.name': option })
+    .select('option_collection_hash')
+    .toArray();
+  const temp = toPairs(signatures);
```
Comment (armenzg) [inline src/utils/perfherder/subbenchmarks.jsx]: Do you need to create the `temp` variable?

File: src/vendor/perf-goggles.js [hunk 5]
```diff
@@ -11,19 +11,19 @@ export const signaturesUrl = (project = PROJECT) =>
 
 const dataPointsEndpointUrl = (project = PROJECT) =>
   `${TREEHERDER}/api/project/${project}/performance/data/`;
-const platformSuitesUrl = ({ frameworkId, platform, project }) =>
+const platformSuitesUrl = ({ framework, platform, project }) =>
```
Comment (armenzg) [inline src/vendor/perf-goggles.js]: Do you have plans to upstream this vended module?
I would like to go back to the published library.
I can show you how to make local changes to perf-goggles while you work on Fx health.

File: README.md (hunk 1 above)
Comment (klahnakoski) [reply README.md]: Treeherder has no standard:  It uses [frameworkId](https://github.com/mozilla/treeherder/search?q=frameworkId&unscoped_q=frameworkId), in code, [framework](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/963d452a7517b5db2c3bf355ea0fc4427ea1220e/src/vendor/perf-goggles.js#L17) in url parameters, and [framework_id](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/336/files#diff-e382eb877fac5502593e9b6b96d9b488R32) in JSON response objects.

File: src/vendor/perf-goggles.js (hunk 5 above)
Comment (klahnakoski) [reply src/vendor/perf-goggles.js]: yes, I would like to upstream this.  I have a local fork of perf-goggles that synchronized with this file.

**Comment by klahnakoski** [comment]
@armenzg ready for another review

**Comment by klahnakoski** [comment]
> Please squash commits into either one or join the ones that make sense joining. Also please have a narrative covering what is being fixed.

We can not squash if we are to be using branches of branches: they have important information to properly merge branches back into master.  Squashing and rebasing destroys information: The parents of a merge can work properly, while the merge does not.  Squashing looses that information.

**Comment by armenzg** [comment]
In that case, could you please use `git rebase -i HEAD~<number>` to edit history where possible?
I've succesfully rebased different branches from one to another even with rewriting history.

**Pull request merged**